src/js/loading-spinner.js

```diff
@@ -17,10 +18,20 @@ class LoadingSpinner extends Component {
    *         The dom element that gets created.
    */
   createEl() {
+    const isAudio = this.player_.isAudio();
+    const playerType = this.localize(isAudio ? 'Audio Player' : 'Video Player');
+
     return super.createEl('div', {
       className: 'vjs-loading-spinner',
       dir: 'ltr'
-    });
+    }, {
+      role: 'alert'
+    }, [
+      dom.createEl('div', {
```

Comment:
I'm not seeing the `innerHTML` in the live example: https://deploy-preview-4916--videojs-docs.netlify.com/test-example/

Also, I think it would be better to use a `span` here as it also patches other `vjs-control-text` uses.